docs/src/man/data.md

```diff
@@ -45,3 +45,12 @@ jq '.components.Device.Injection.Generator.ThermalGen.ThermalDispatch | .[] | se
 ```
 jq -r '["name", "econ.capacity"], (.components.Device.Injection.Generator.ThermalGen.ThermalDispatch | .[] | [.name, .econ.capacity]) | @tsv' system.json
 ```
+
+## View the forecast types and initial_time values.
```

Comment:
Note that I added this after your approval. You may not have used this functionality yet...it can really help inspect the data.